### DISCUSSION TITLE
Ability to parse placeholders in the autorankup section

### DISCUSSION THREAD

**Discussion by lochlanmcelroy** [Zrips/CMI#7922] [Feature Requests]
Hoping to have the ability to make CMI parse a placeholder in the autorankup section, instead of just being limited to its preset ones. It would hopefully look something like this:

 StatsRequirements:
  - %quests_completed%:3
  - %parkour_score%:25
  
  Or something similar!

**Comment by mrfloris** [comment]
https://github.com/Zrips/CMI/discussions/4785

**Reply by lochlanmcelroy** [reply]
My bad, it was a few years old so I didnt see it
